components/ProfileForm/index.js

```diff
@@ -0,0 +1,166 @@
+import styled from "styled-components";
+import { useEffect } from "react";
+
+export default function ProfileForm({ setFormData, formData }) {
+  //Handle input change
+  const handleInputChange = (event) => {
+    const { name, value } = event.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  //Handle form submit
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(formData);
+  };
+
+  useEffect(() => {
+    // Function to convert the date to the desired format: 12.12.1981
+    const formatDate = (date) => {
+      const day = date.getDate().toString().padStart(2, "0");
+      const month = (date.getMonth() + 1).toString().padStart(2, "0");
+      const year = date.getFullYear();
+      return `${day}.${month}.${year}`;
+    };
+
+    // Current Date
+    const currentDate = new Date();
+
+    // Save date in the desired format
+    setFormData({ ...formData, entryDate: formatDate(currentDate) });
+  }, []);
+
+  return (
+    <ProfileFormDiv onSubmit={handleSubmit}>
+      <div className="column-left">
```

Comment:
Here for example youre using css instead of styled-components

Reply:
Also im missing the vercel deployment, but for the next step you have to do this anyway ;)

Reply:
@HannesOster thanks for your reply! Problem by Vercel is that it render the index. Here i deploy the profile page, you can find it her in the deployment -> https://aipersonaltrainer-llzrkur8v-oliverhees.vercel.app/profile